dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -48,3 +48,13 @@ properties:
     pinctrl-0:
       type: phandles
       required: true
+
+    bank:
+      type: int
+      description: qspi bank pins to configure
+      required: false
+      enum:
+        - 1
+        - 2
+      default: 1
```

Comment:
All uses of default must be justified. See https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings.